custom_filter_groupby_visibility_control/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,4 @@
+This module allows restricting access to the “Add Custom Group” and “Add Custom Filter” options in the search view.
+
+It introduces a user group: “Hide Custom Filter and Group By”.
+Users who belong to this group will not see the “Add Custom Filter” and “Add Custom Group” buttons.
```

Comment:
Doesn't it go against how access rights usually work (i.e., you give permissions, instead of taking them away)?